lib/util.js

```diff
@@ -138,11 +139,16 @@ function cleanUpAssertion(element) {
 }
 
 function getAssertionComponents(element) {
-  let matched = element.match(/(.*)\/\/\s(\w+-?\w+?)(:\s(.+))?;/);
+  const regEx =/(.*)\/\/\s*(?:([a-z\-]+:|throws:?))([^;]+)?(?:.*)/i;
```

Comment:
as a good RegExp best practice lets root this RegExp at the start of the line, by beginning it with `^~`